x/params/types/proposal.go

```diff
@@ -0,0 +1,121 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+const (
+	// ProposalTypeChange defines the type for a ParameterChangeProposal
+	ProposalTypeChange = "ParameterChange"
+)
+
+// Assert ParameterChangeProposal implements govtypes.Content at compile-time
+var _ govtypes.Content = ParameterChangeProposal{}
+
+func init() {
+	govtypes.RegisterProposalType(ProposalTypeChange)
```

Comment:
@rigelrozanski I think it's best to keep it local to where the type is defined. I believe this is best because:

* Less to worry about/less error prone -- you just need to import the type, which you'll do when you register the handler
* Ease of use -- if I want to implement integration tests (or anything else really) incorporating such custom proposals I must be sure to manually register OR call some public function from `app.go`

Reply:
👍